### PR TITLE
CHANGELOG: Document changes since 0.14.0

### PR DESCRIPTION
**Pull request by wking**
Through 0d891e1119 (#1446).  Also rebuild the dependency SVG.

CC @abhinavdahiya

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,98 @@ All notable changes to this project will be documented in this file.
...
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## 0.15.0 - 2019-03-25
+
+### Added
+
+- We now initialize TLS certificates for etcd metrics over TCP on port
+  9979.  We also store etcd certificates in various `kube-system`
+  Secrets and ConfigMaps.
+- The Kubernetes client is not extracted from the release image's
```
Comment (crawford) [inline CHANGELOG.md]: `s/not/now/`?

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,98 @@ All notable changes to this project will be documented in this file.
...
+  to more closely align with `Network.config.openshift.io`:
+    - `serviceCIDR` is now `serviceNetwork`.
+    - `clusterNetworks` is now `clusterNetwork`.
+    - `type` is now `networkType`.
+    - `hostSubnetLength` is now `hostPrefix`.
+    `v1beta3` is deprecated and will be removed in a future release.
+- On AWS and OpenStack, ports 9000 through 9999 are now open for UDP.
+  They had been open for TCP since 0.4.0, with a bugfix for 9990 ->
+  9999 in 0.13.0.
+- On AWS, we now created network interfaces for the control-plane
```
Comment (crawford) [inline CHANGELOG.md]: s/created/create/

File: CHANGELOG.md (hunk 1 above)
Comment (wking) [reply CHANGELOG.md]: > s/not/now/?

Fixed with 4cc437ae9 -> 69d3eb17a.

File: CHANGELOG.md (hunk 2 above)
Comment (wking) [reply CHANGELOG.md]: > s/created/create/

Fixed with 4cc437a -> 69d3eb1.

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,98 @@ All notable changes to this project will be documented in this file.
...
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## 0.15.0 - 2019-03-25
+
+### Added
+
+- We now initialize TLS certificates for etcd metrics over TCP on port
+  9979.  We also store etcd certificates in various `kube-system`
+  Secrets and ConfigMaps.
+- The Kubernetes client is now extracted from the release image's
```
Comment (abhinavdahiya) [inline CHANGELOG.md]: This is not clear that MCO uses those images.

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,98 @@ All notable changes to this project will be documented in this file.
...
+  `kube-client-agent` reference, replacing
+  `quay.io/coreos/kube-client-agent`.
+- CRI-O's pause image is now extracted from the release image's `pod`
```
Comment (abhinavdahiya) [inline CHANGELOG.md]: This doesn't make it clear that MCO uses these for control plane and compute nodes. Bootstrap node doesn't use it...

File: CHANGELOG.md (hunk 3 above)
Comment (wking) [reply CHANGELOG.md]: > This is not clear that MCO uses those images.

Suggested wording?

File: CHANGELOG.md [hunk 5]
```diff
@@ -4,6 +4,98 @@ All notable changes to this project will be documented in this file.
...
+    - `type` is now `networkType`.
+    - `hostSubnetLength` is now `hostPrefix`.
+    `v1beta3` is deprecated and will be removed in a future release.
+- On AWS and OpenStack, ports 9000 through 9999 are now open for UDP.
+  They had been open for TCP since 0.4.0, with a bugfix for 9990 ->
+  9999 in 0.13.0.
+- On AWS, we now create network interfaces for the control-plane
+  nodes explicitly, which allows for faster resource-creation time by
+  allowing greater parallelization.
+- On AWS, we now ask the machine-API operator to use
```
Comment (abhinavdahiya) [inline CHANGELOG.md]: nit: we ask doesn't sound great. Actually machinesets use a secret that mao sync to `openshift-machine-api`

File: CHANGELOG.md (hunk 4 above)
Comment (wking) [reply CHANGELOG.md]: > This doesn't make it clear that MCO uses these for control plane and compute nodes.

Reworded with 65607bb98 -> 620c0c0f6.  How does that look?

File: CHANGELOG.md (hunk 5 above)
Comment (wking) [reply CHANGELOG.md]: > nit: we ask doesn't sound great.

Suggested wording for an entry covering #1281?

File: CHANGELOG.md [hunk 6]
```diff
@@ -4,6 +4,98 @@ All notable changes to this project will be documented in this file.
...
+- On AWS, the bootstrap machine is now created in the first public
+  subnet, restoring SSH and journald access, and fixing a bug from
+  0.14.0.
+- On AWS, the Kubernetes API load balancers now use `/readyz` instead
+  of `/healthz` for health checks, which allows for more graceful
+  control-plane rotation.
+- On AWS, `destroy cluster` has some fixes for:
+    - Removing snapshots associated with copied AMIs, fixing a bug
+      from 0.14.0.
+    - Network-interface deletion, where we now remove all network
```
Comment (abhinavdahiya) [inline CHANGELOG.md]: Not sure if AWS docs uses hyphen. https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-eni.html

Comment (wking) [reply CHANGELOG.md]: > Not sure if AWS docs uses hyphen.

Fixed with 620c0c0f6 -> bb387148c.

**Comment by wking** [comment]
[e2e-aws][1]:

```
level=fatal msg="failed to initialize the cluster: Working towards 0.0.1-2019-03-26-043823: 99% complete: timed out waiting for the condition"
```

/retest

[1]: https://openshift-gce-devel.appspot.com/build/origin-ci-test/pr-logs/pull/openshift_installer/1462/pull-ci-openshift-installer-master-e2e-aws/4635

**Pull request merged**